onadata/apps/api/tests/viewsets/test_organization_profile_viewset.py

```diff
@@ -28,6 +29,13 @@ def setUp(self):
             'patch': 'partial_update',
         })
 
+    def tearDown(self):
+        """
+        Specific to clear cache between tests
+        """
+        super(self.__class__, self).tearDown()
```

Comment:
Codacy found an issue: [Bad first argument 'self.__class__' given to super()](https://app.codacy.com/gh/onaio/onadata/pullRequest?prid=6016417)